setup-jest.js

```diff
@@ -0,0 +1 @@
+require('@inrupt/jest-jsdom-polyfills');
```

Comment:
please make this into a TypeScript file and use `import`.

Reply:
Done in 7d1cffa8